src/Numerics/Mesh/Topologies.jl

```diff
@@ -171,6 +173,40 @@ struct BoxElementTopology{dim, T, nb} <: AbstractTopology{dim, T, nb}
     """
     bndytoface::NTuple{nb, Array{Int64, 1}}
 
+    """
+    Element to locally unique vertex number; `elemtouvert[v,e]` is the `v`th vertex 
+    of element `e`
+    """
+    elemtouvert::Union{Array{Int64, 2}, Nothing}
+
+    """
+    Vertex connectivity information for direct stiffness summation; 
+    `vtconn[lvt1, lelem1, lvt2, lelem2, ....]`
+    """
+    vtconn::Union{AbstractArray{Int64, 1}, Nothing}
+
+    """
+    Vertex offset for vertex connectivity information
+    """
+    vtconnoff::Union{AbstractArray{Int64, 1}, Nothing}
+
+    """
+    Face connectivity information for direct stiffness summation; 
+    `fcconn[ufc][lfc1, lelem1, lfc2, lelem2, ordr]`
+    """
+    fcconn::Union{AbstractArray{Int64, 2}, Nothing}
+
+    """
+    Edge connectivity information for direct stiffness summation; 
+    `edgconn[uedg][ledg1, orient, lelem1, ledg2, orient, lelem2, ...]`
```

Comment:
```suggestion
    Edge connectivity information for direct stiffness summation; 
    ```
    edgconn[edgconnoff[i]:edgconnoff[i+1]-1] == [ledg1, orient1, lelem1, ledg2, orient2, lelem2, ...]
    ```
    where `ledg1` is the element-local edge number, `orient1` is orientation (1 or 2) of element `lelem1`.
```

Reply:
Add reference for edge numbering.